### ISSUE TITLE
Can I reply to a conversation with a button click?

### ISSUE THREAD

**Issue by ManuZenou** [howdyai/botkit#727]
I have a bot that starts a conversation, and I want to add a "never mind" button that "exists" the conversation and stops it.
what is the best way to do it?

Thanks in advanced.

**Comment by peterswimm** [comment]
For which platform are you running your bot?

**Comment by ManuZenou** [comment]
slack

**Comment by peterswimm** [comment]
Take a look at interactive messages:
https://github.com/howdyai/botkit/blob/master/readme-slack.md#message-buttons

**Comment by ManuZenou** [comment]
exactly what I needed (the second example)

**Comment by peterswimm** [comment]
Great! Have fun!

Closing for now...

**Comment by ManuZenou** [comment]
another question,
I have integrated the [interactive message in conversation](https://github.com/howdyai/botkit/blob/master/readme-slack.md#using-interactive-messages-in-conversations), and now i'm facing another issue. 

as a part of the implementation, I had to add a `interactive_replies: true,` parameter to by bot init function. ever since, every "regular" (not a part of a conversation) button my bot has is also treated as text message, due to this snippet on the slackbot.js:
`if (configuration.interactive_replies) {
            message.type = 'message';
            slack_botkit.receiveMessage(bot, message);
        }`

this causes any button interaction to be treated as a message, and my '*' rule catches it as if it was a regular command as well.

any ideas?
